### PR TITLE
IPFSPAN Projects join slingshot

### PR DESCRIPTION
**Pull request by IPFSPAN**
IPFSPAN Projects join slingshot

### CODE REVIEW

**Comment by dineshshenoy** [comment]
Thanks for submitting an application.  Our team will validate for completeness and then assign reviewers from the competitions review panel.

**Comment by jasonleyser** [comment]
Hi @IPFSPAN , Thanks for the submission. Can you add your slack username to the PR please, and then we can accept this application.

File: participants/IPFSPAN.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+people use this app to store, send, and receive digital currency,or save and share files data access on desktop, mobile and web. 
+
+(2) Link to Project UI
+
+https://ipfspan.com
+
+(3) What does your application/UI do?
+
+IPFSPAN is a Filecoin wallet and network cloud storage Drive build on IPFS.
```
Comment (GregTheGreek) [inline participants/IPFSPAN.md:21]: Are you only storing in IPFS?

**Comment by IPFSPAN** [comment]
Yes，only storing in IPFS  So the project name is IPFSPAN------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;&quot;Gregory&amp;nbsp;Markou&quot;<notifications@github.com&gt;
发送时间:&nbsp;2020年9月25日(星期五) 晚上11:44
收件人:&nbsp;&quot;filecoin-project/slingshot&quot;<slingshot@noreply.github.com&gt;;
抄送:&nbsp;&quot;IPFSPAN&quot;<8128661@qq.com&gt;;&quot;Mention&quot;<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [filecoin-project/slingshot] IPFSPAN Projects join slingshot (#34)

**Pull request merged**